pkg/didcomm/protocol/issuecredential/models.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package issuecredential
+
+// ProposeCredential an optional message sent by the potential Holder to the Issuer
+// to initiate the protocol or in response to a offer-credential message when the Holder
+// wants some adjustments made to the credential data offered by Issuer.
+type ProposeCredential struct {
+	Type string `json:"@type"`
+	ID   string `json:"@id"`
```

Comment:
These headers aren’t defined by messenger?

Reply:
We cannot remove `@type`. The `ID` looks redundant.

Reply:
Removed @id